airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/streams/base_streams.py

```diff
@@ -294,7 +281,7 @@ def state(self) -> Mapping[str, Any]:
         """State getter, get current state and serialize it to emmit Airbyte STATE message"""
         if self._cursor_values:
             result_state = {account_id: {self.cursor_field: cursor_value} for account_id, cursor_value in self._cursor_values.items()}
-            result_state["include_deleted"] = self._include_deleted
+            result_state["filter_statuses"] = self._filter_statuses
```

Comment:
We need avoid braking change and if user has included_deleted parameter use it.

Reply:
I understand the main idea, but for users who have already set up synchronization, this will be a breaking change, requiring them to reconfigure their connections.

We aim to avoid this inconvenience. Therefore, let's revise the logic so that if the `include_deleted` parameter exists, we will utilize the old logic. However, if the `filter_statuses` list is set up, we will exclusively use this list.

Reply:
I understand the main idea, but for users who have already set up synchronization, this will be a breaking change, requiring them to reconfigure their connections.

We aim to avoid this inconvenience. Therefore, let's revise the logic so that if the include_deleted parameter exists, we will utilize the old logic. However, if the filter_statuses list is set up, we will exclusively use this list.

Reply:
It is breaking change - who do not need new functionality should stay on older version. I will not over-engineer for backward compatibility.

Reply:
The logic having empty selected statuses works even in case if include_deleted were set previously, but you have insisted to use ALL. So now we have real breaking changes (previously it was only formally breaking) and need to deal with logic to decide either use previous flag or force to use new filter_statuses.